### DISCUSSION TITLE
关于free_quota转换为整数类型时出现了问题

### DISCUSSION THREAD

**Discussion by youhaozhao** [BurnHair/docs#156] [技术支持]
<img width="300" alt="截屏2024-07-10 上午12 48 33" src="https://github.com/BurnHair/docs/assets/128596662/64cd34d4-1095-414f-af90-e95cab70090e">
报错如上，我的额度早上剩下22刀，过去一共用掉了7刀，现在显示“用户额度已耗尽”，是这里出了问题吗？
<img width="1007" alt="截屏2024-07-10 上午12 50 45" src="https://github.com/BurnHair/docs/assets/128596662/5f5a44a3-a3ad-4ac1-be18-923f8dc26ddd">

**Comment by BennyThink** [comment]
在修了在修了.jpg

**Comment by BennyThink** [comment]
应该好了！

**Reply by youhaozhao** [reply]
嗯嗯好了，感谢您
